diff --git a/node_modules/@chainsafe/libp2p-quic/dist/src/listener.js b/node_modules/@chainsafe/libp2p-quic/dist/src/listener.js
index 0ea2a70..c232cb7 100644
--- a/node_modules/@chainsafe/libp2p-quic/dist/src/listener.js
+++ b/node_modules/@chainsafe/libp2p-quic/dist/src/listener.js
@@ -31,7 +31,7 @@ export class QuicListener extends TypedEventEmitter {
         this.log('new');
     }
     updateAnnounceAddrs(addrs) {
-        throw new Error('Method not implemented.');
+
     }
     getAddrs() {
         if (this.state.status === 'listening') {
